src/Actions/ManagesIssues.php

```diff
@@ -14,12 +14,19 @@ trait ManagesIssues
      * Get a list of of issues.
      *
      * @param string $projectId
+     * @param int $limit
+     * @param int $page
      *
      * @return Issue[]
+     * @throws \TestMonitor\Mantis\Exceptions\InvalidDataException
      */
-    public function issues($projectId)
+    public function issues($projectId, $limit = 25, $page = 1)
```

Comment:
I'd use Mantis' default, limit = 50

Reply:
And we use strong typing in the other clients, perhaps here as well?

Reply:
If we follow strict API implementation in this case $projectId should be optional.

Reply:
Agreed